Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/grenade.yml

```
@@ -41,8 +45,10 @@
   parent: BaseBulletIncendiary
   components:
   - type: Sprite
-    sprite: Objects/Weapons/Guns/Projectiles/projectiles2.rsi
-    state: buckshot-flare
+    sprite: _Sunrise/Objects/Weapons/Projectiles/shrapnel.rsi #Sunrise-Start
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)